components/aws/sagemaker/common/_utils.py

```diff
@@ -111,6 +110,9 @@ def create_training_job_request(args):
     ### Update input channels, must have at least one specified
     if len(args['channels']) > 0:
         request['InputDataConfig'] = args['channels']
+        for i in range(1, 9):
```

Comment:
how did we get the value '9' - please move it to a constant

Reply:
avoid the magic numbers :)